Content.Server/Shuttles/Systems/ShuttleSystem.FasterThanLight.cs

```diff
@@ -637,11 +637,9 @@ private void DoTheDinosaur(TransformComponent xform)
             {
                 if (!_statusQuery.TryGetComponent(child, out var status))
                     continue;
-                
-                if (HasComp<FTLKnockdownImmuneComponent>(child)) // Frontier: NPC knockdown immunity
-                    continue; // Frontier: NPC knockdown immunity
 
-                _stuns.TryParalyze(child, _hyperspaceKnockdownTime, true, status);
+                if (!HasComp<FTLKnockdownImmuneComponent>(child)) // Frontier: FTL knockdown immunity
+                    _stuns.TryParalyze(child, _hyperspaceKnockdownTime, true, status);
```

Comment:
Should FTL-immune players be tossed if standing on space?  Should NPCs?

Reply:
Yes and yes.
Stronger yeet.